app/views/tasks/_in_progress_tasks.html.erb

```diff
@@ -6,7 +6,7 @@
     <div class="panel-body">
       <% tasks.each do |task| %>
           <ul class="list-group">
-            <li class="list-group-item"><%= task.name %>
+            <li class="list-group-item"><%= task.name %>(<%= link_to (task.user.initials), profile_path(task.user.profile) %>)
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/11/files#r28692964)